ss58-registry.json

```diff
@@ -910,6 +910,15 @@
       "standardAccount": "*25519",
       "website": "https://tidefi.com"
     },
+    {
+      "prefix": 7013,
+      "network": "gm",
+      "displayName": "GM",
+      "symbols": ["FREN", "GM", "GN"],
+      "decimals": [12, 0, 0],
```

Comment:
Why are these decimals 0?

Reply:
Because these tokens have no decimals, while for FREN 1 unit is 1000000000000, for GM and GN 1 unit is 1